pkg/util/cmdutil/console.go

```diff
@@ -93,3 +94,110 @@ func RemoveTralingNewline(s string) string {
 
 	return s
 }
+
+// TableRow is a row in a table we want to print.  It can be a series of a columns, followed
+// by an additional line of information.
+type TableRow struct {
+	Columns        []string
+	AdditionalInfo string
+}
+
+// PrintTable prints a grid of rows and columns.  Width of columns is automatically
+// determined by the max length of the items in each column.
+func PrintTable(table []TableRow) {
+	if len(table) == 0 {
+		return
+	}
+
+	maxColumnWidths := make([]int, len(table[0].Columns))
+	for i := range maxColumnWidths {
+		maxColumnWidths[i] = -1
+	}
+
+	PrintTableEx(table, maxColumnWidths, "  " /*columnGap*/)
+}
+
+const ellipses = "..."
+
+// PrintTableEx prints a grid of rows and columns.  Width of columns is provided.  Column values
+// longer than this width will be automatically truncated.  Use -1 to indicate no max for a
```

Comment:
I don't think we should be truncating these values. Most of the times we print these tables, it's important information, and I think just using as much width as necessary to show them is preferred.

Reply:
Okey dokey.

Reply:
If we did this before, or there was a specific case you added it for, please do let me know ... My bias is just not to truncate since it loses valuable information and may lead to users getting annoyed and needing to `--json` just to see the full info.

Reply:
I was actually misunderstanding the previous system. We had some hard coded lengths.  I thought those acted as maximums.  instead, tehy were minimums, and we'd always print everythng.  So i've updated the pr to match that behavior.  Thanks for catching!